### PR TITLE
Fix references, WebKit dialog in playground

### LINKED ISSUE #1322: new_embeddings_demo.html fails to load in Safari

### LINKED ISSUE #1356: New playground should warn about Safari
Because we're still stuck with the DDC/Safari issue, the new playground should display the same message the new embed uses when:

* Someone arrives at the playground with a gist ID, sample ID, or API Doc sample in the URL that will require the playground to be in Flutter mode.
* They use the Type/Layout dropdown to enter the Flutter layout.

One popup per pageload is all we'd want to throw at them, so a flag in the page indicating whether it's been shown is probably a good idea as well.

### PR DESCRIPTION
**Pull request by johnpryan**
- Fix script references
- show WebKit dialog in playground
- prevent auto-run in WebKit.

closes #1356
closes #1322

### CODE REVIEW

File: lib/util/detect_safari.dart [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<p>
+It looks like you're using a WebKit-based browser (such as Safari). There's
+currently an issue with the way DartPad and WebKit's JavaScript parser interact
+that could cause up to a thirty second delay the first time you execute Flutter
+code in DartPad. This is not an issue with Dart or Flutter itself, and we're
+working with the WebKit team to resolve it.
+</p>
+<p>
+In the meantime, it's possible to avoid the delay by using one of the other
+major browsers, such as Firefox, Edge (dev channel), or Chrome.
```
Comment (domesticmouse) [inline lib/util/detect_safari.dart]: Suggestion: turn these references into links to install pages:
 - https://www.mozilla.org/firefox/new/
 - https://www.microsoftedgeinsider.com/en-au/download
 - https://www.google.com/chrome/

File: lib/util/detect_safari.dart [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import 'dart:html';
```
Comment (RedBrogdon) [inline lib/util/detect_safari.dart]: Copyright header needed.

Comment (RedBrogdon) [inline lib/util/detect_safari.dart]: Do you mind renaming this file to `detect_webkit`? I've been trying wherever possible to make sure no one thinks we're trying to throw Safari under the bus.

File: lib/util/detect_safari.dart (hunk 1 above)
Comment (johnpryan) [reply lib/util/detect_safari.dart]: good idea, added.

File: lib/util/detect_safari.dart (hunk 2 above)
Comment (johnpryan) [reply lib/util/detect_safari.dart]: fixed, thanks

Comment (johnpryan) [reply lib/util/detect_safari.dart]: yep, fixed.

**Pull request merged**